internal/pkg/resource/resource.go

```diff
@@ -29,6 +29,7 @@ const (
 	Pipeline              = "pipeline"
 	SchemaExporter        = "schema exporter"
 	SchemaRegistryCluster = "Schema Registry cluster"
+	SchemaRegistryConfig  = "Schema Registyr config"
```

Comment:
```suggestion
	SchemaRegistryConfiguration  = "Schema Registry configuration"
```

Reply:
`config` is how the route is referenced in docs, and also used in descriptions. wdyt?

Reply:
`config` is a common shorthand, but it doesn't hurt to use the full word and it's a bit more explicit / professional looking, IMO.